.cmake-format.json

```diff
@@ -0,0 +1,14 @@
+{
+  "line_width": 80,
+  "dangle_parens": false,
```

Comment:
this probably should be true

Reply:
I'm going to have to redo this PR because of the comment reflow, I'll change this over then